File: Day-01_Python3/part_one.py
input = open('input.txt', 'r').read()
data = input.split('\n\n')

output = 0
for x in data:
    temp = 0
    x = x.split()
    for i in x:
        temp += int(i)
    if temp > output:
        output = temp

print(output)

File: Day-04_JavaScript/part-one.js
// Read data
var fs = require("fs");
var readFile = function(path) {
    return fs.readFileSync(path).toString();
};

// Evaluate lines
var sum = 0;
function process_line(line) {
    ranges = line.replaceAll('-', ',').split(',').map(Number);
    if ((ranges[0] <= ranges[2] && ranges[1] >= ranges[3]) 
        || (ranges[0] >= ranges[2] && ranges[1] <= ranges[3]))
    {
            sum += 1;
    }
};

var lines = readFile('input.txt').split('\n');
lines.forEach(process_line);
console.log(sum);